spring-5-security/src/main/resources/application.properties

```diff
@@ -2,4 +2,10 @@ server.port=8081
 
 logging.level.root=INFO
 
-logging.level.com.baeldung.dsl.ClientErrorLoggingFilter=DEBUG
\ No newline at end of file
+logging.level.com.baeldung.dsl.ClientErrorLoggingFilter=DEBUG
+
+spring.datasource.driver-class-name = org.hsqldb.jdbc.JDBCDriver
```

Comment:
if we have added these we should not be needing persistence.properties, also please check the properties in my opinion for embedded h2 server we should not be needing these properties at all